src/unstract/adapters/vectordb/helper.py

```diff
@@ -28,20 +29,19 @@ def test_vector_db_instance(
             if vector_store is None:
                 return False
 
-            # For custom embedding args will be:
-            #     embed_model - InstructorEmbeddings(embed_batch_size=2)
-            #     chunk_size - 512
-            #     llm=None
-            service_context = ServiceContext.from_defaults(
-                llm=None, embed_model=None
-            )
-
             storage_context = StorageContext.from_defaults(
                 vector_store=vector_store
             )
             local_path = f"{os.path.dirname(__file__)}/samples/"
+            # Using mock llm and embedding here.
+            # For custom embedding args will be:
+            #     embed_model - InstructorEmbeddings(embed_batch_size=2)
+            #     chunk_size - 512
+            #     llm=None
+            llm = MockLLM()
```

Comment:
QQ: @gaya3-zipstack will the token usage for such cases also be tracked?

Reply:
Point. I have not attached callback manager here. It may do it once I attach one.

Reply:
Will add this in the next iteration as I will need to test again with that. ANyway token usage here even if recorded is not against any adapter that is configured explicitly.

Reply:
> ANyway token usage here even if recorded is not against any adapter that is configured explicitly.

Yeah you're right.
Please add a TODO comment to not lose track of it, we can discuss further - anyway my knowledge on these MockLLM or MockEmbedding objects is limited. Do these require any keys? Maybe it doesn't need to be considered.
CC: @johnyrahul @arun-venkataswamy